index.bs

```diff
@@ -1367,7 +1367,7 @@ Build a composed graph up to a given output operand into a computational graph a
     1. Let |operators| be a new empty [=/set=].
     1. Let |inputs| be a new empty [=/set=].
     1. Let |queue| be a new [=queue=] containing |outputs|'s [=map/values=].
-    1. While |queue| is not [=queue/empty=]:
+    1. While |queue| [=queue/is empty|is not empty=]:
```

Comment:
I'm perplexed by the new wording. Is this saying when the queue is either empty _or_ not empty?

Reply:
So Bikeshed's autolink syntax is `term|text` so you can override the term with custom text. The "queue/is empty" part is the (scoped) link to the term, the "is not empty" part after the pipe is what will show in the spec.

It's definitely awkward... and the good news is that Infra **_does_** define "is not empty" as an alias for "is empty" so this can be simplified to just `[=queue/is not empty=]`.

I'll update the PR next week unless someone beats me to it.

Reply:
> so you can override the term with custom text.

👍. So then what's still bending my brain 🙃 is the disagreement between the two sides of the `|`, one linking to "is empty" but the other stating the *opposite* "is not empty" (kinda like saying `[=queue/is red|is blue=]`), whereas if it had been `While [queue] not [=queue/is empty=]`, it would agree. I'll update it.

Reply:
LGTM!